src/components/Table/Table.tsx

```diff
@@ -2,22 +2,28 @@ import React from "react";
 
 import { Total } from "../Total/Total";
 import { testIds } from "./testIds";
-import { Buttons } from "../Buttons/Buttons";
+import { Button } from "../Button/Button";
 
 import table from "./Table.module.css";
 
 type IExpance = {
   name: string;
   amount: number;
   id: string;
+  withConfirm?: boolean;
 };
 
 type IProps = {
   operationList: IExpance[];
-  removeItem: Function;
+  removeItem: (id: string) => void;
+  withConfirm?: boolean;
 };
 
-export const Table = ({ operationList, removeItem }: IProps) => {
+export const Table = ({
+  operationList,
+  removeItem,
+  withConfirm = true,
```

Comment:
this is probably bad practice, but did not understand how to do it better

Reply:
```suggestion

```